dockertest/output/dockerinfo.py

```diff
@@ -0,0 +1,104 @@
+"""
+Handler for 'docker info'. Runs command, returns output in
+machine-friendly form.
+"""
+
+import collections
+import subprocess
+
+
+class DockerInfo(collections.MutableMapping):
+    """
+    Parser of 'docker info' output
+    """
+
+    def __init__(self, info_string=None, docker_path=None):
+        if info_string is None:
+            if docker_path is None:
+                docker_path = 'docker'
+            info_string = subprocess.check_output(docker_path + ' info',
+                                                  shell=True,
+                                                  close_fds=True)
+        self.info_string = info_string
+        self.table = self._build_table(info_string)
```

Comment:
Uggg, PEBUAK, my bad.  You **SHOULD** call ``super(...)__init__()`` (first draft did not).